cookbooks/aws-parallelcluster-config/recipes/fsx_mount.rb

```diff
@@ -38,32 +44,64 @@
   dns_name = if fsx_dns_name && !fsx_dns_name.empty?
                fsx_dns_name
              else
-               # Hardcoded DNSname only valid for filesystem created after Mar-1 2021
-               # For older filesystems, DNSname needs to be retrieved from FSx API
+               # Hardcoded DNS name only valid for lustre file systems created after Mar-1 2021
+               # For older file systems, DNS names need to be retrieved from FSx API
+               # For Ontap and OpenZFS file systems, the DNS name is always passed from CLI.
```

Comment:
What do you mean with "passed *from* CLI"? Do you mean that you pass it *to* the CLI?
Furthermore, the comment is a bit misleading, at least to me, because it makes me think that I should expect a call to an API to retrieve the dns name, whereas the code actually hard code the DNS name given the FS id and region. May you please clarify?

Reply:
Thanks, I improved the comment